mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -630,3 +633,70 @@ Feature: Loans tests
+    * def itemId = postItemResponse.response.id
+
+    # post a group and user
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostGroup')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserBarcode: #(extUserBarcode) }
+
+    # post owner, manual charge and payment method
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostOwner')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostManualCharge')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostPaymentMethod')
+
+    # checkOut the item
+    * def checkOutResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostCheckOut') { extCheckOutUserBarcode: #(extUserBarcode), extCheckOutItemBarcode: #(extItemBarcode) }
+    * def loanId = checkOutResponse.response.id
+    And match checkOutResponse.response.item.id == itemId
+    And match checkOutResponse.response.item.status.name == 'Checked out'
+    And match checkOutResponse.response.loanDate == '#present'
+    And match checkOutResponse.response.dueDate contains extDueDate
+
+    # checkIn the item
+    * def postCheckInResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@CheckInItem') { itemBarcode: #(extItemBarcode), extCheckInDate: #(extCheckInDate) }
+    And match postCheckInResponse.response.loan.id == loanId
+    And match postCheckInResponse.response.loan.status.name == 'Closed'
+    And match postCheckInResponse.response.item.id == itemId
+    And match postCheckInResponse.response.item.status.name == 'Available'
+    And match postCheckInResponse.response.loan.returnDate contains extCheckInDate
+    And match postCheckInResponse.response.loan.dueDate contains extDueDate
+
+    # get overdue fine by loan id
+    Given path 'accounts'
+    And param query = 'loanId==' + loanId
+    When method GET
+    Then status 200
+    And match response.totalRecords == 1
+    And match response.accounts[0].status.name == 'Open'
+    And match response.accounts[0].feeFineType == 'Overdue fine'
+    And def accountId = response.accounts[0].id
+    And def overdueFine = response.accounts[0].amount
+
+    # check-pay and verify that the fine is possible to be paid
+    Given path 'accounts/' + accountId + '/check-pay'
+    And request { amount: '#(overdueFine)' }
+    When method POST
+    Then status 200
+    And match response.allowed == true
+
+    # pay overdue fine and verify that the fine is billed fully
+    * def postPayResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostPay') { amount: #(overdueFine) }
+    And match postPayResponse.response.feefineactions[0].typeAction == 'Paid fully'
+    And match postPayResponse.response.feefineactions[0].amountAction == overdueFine
+    And match postPayResponse.response.feefineactions[0].accountId == accountId
```

Comment:
```suggestion
    * def postPayResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostPay') { amount: #(overdueFineAmount) }
    * def feefineactionInResponse = postPayResponse.response.feefineactions[0]
    And match feefineactionInResponse .typeAction == 'Paid fully'
    And match feefineactionInResponse .amountAction == overdueFineAmount
    And match feefineactionInResponse .accountId == accountId
```
add empty line at the end

Reply:
In addition add three files ( owner-entity-request.json pay-entity-request.json payment-method-entity-request.json ) under folder "feefine" which is will be located as the same level as folders "instance", "item" etc

Reply:
done